platform/process/tasks/ABProcessTaskServiceAccountingBatchProcessing.js

```diff
@@ -205,6 +206,8 @@ module.exports = class AccountingBatchProcessing extends AccountingBatchProcessi
             journalEntry
          )
             .then(() => {
+               if (this.jeStatusField == null) return resolve();
```

Comment:
Question: is this one of those cases where ```.jeStatusField == null``` is a mis configuration?

Reply:
No, that option is optional.